space/space.go

```diff
@@ -54,6 +55,7 @@ type Repository interface {
 	Load(ctx context.Context, ID satoriuuid.UUID) (*Space, error)
 	Delete(ctx context.Context, ID satoriuuid.UUID) error
 	List(ctx context.Context, start *int, length *int) ([]*Space, uint64, error)
```

Comment:
Yeah, I agree with that. I only prefer reviewing these only after we see the UI wireframes.